core/tests/protractor_desktop/preferences.js

```diff
@@ -109,6 +109,21 @@ describe('Preferences', function() {
       'http://localhost:9001/learner_dashboard');
   });
 
+  it('should set the subject interest', function() {
+    users.createUser('nipan@preferences.com', 'nipanPreferences');
```

Comment:
Please avoid putting in your own name anywhere besides TODO comments, and prefer using descriptive names like "creator", "learner", "superUser", etc. instead.